Reading List/Reading List/Model Controllers/BookController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  BookController.swift
+//  Reading List
+//
+//  Created by Richard Gibbs on 5/19/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import Foundation
+
+
+class BookController: Codable {
+    var books: [Book] = []
+    var readBooks: [Book]  {
+        books.filter {
+            $0.hasBeenRead == true
+        }
+    }
+    var unreadBooks: [Book] {
+        books.filter {
+            $0.hasBeenRead == false
+        }
+    }
+    var readingListURL: URL? {
+        
+        let fileManager = FileManager.default
+        let documentsDir = fileManager.urls(for: .documentDirectory, in: .userDomainMask).first
+        let booksURL = documentsDir?.appendingPathComponent("ReadingList.plist")
+        return booksURL
+    }
+    
+    func createBook(title: String, reasonToRead: String, hasBeenRead: Bool) {
+        let book = Book(title: title, reasonToRead: reasonToRead, hasBeenRead: hasBeenRead)
+        books.append(book)
+        saveToPersistentStore()
+    }
+    
+    func deleteBook(book: Book) {
+        if  let index = books.firstIndex(of: book) {
+            books.remove(at: index)
```

Comment:
when the user deletes something, closes the app and comes back to it, should they see the same what they deleted?